bioio/bio_image.py

```diff
@@ -281,36 +333,69 @@ def ends_with_ext(string: str) -> bool:
     @staticmethod
     def _get_reader(
         image: biob.types.ImageLike,
-        reader: biob.reader.Reader,
+        reader: Optional[
+            Union[Type[biob.reader.Reader], Sequence[Type[biob.reader.Reader]]]
+        ],
         use_plugin_cache: bool,
         fs_kwargs: Dict[str, Any],
         **kwargs: Any,
     ) -> Tuple[biob.reader.Reader, Optional[PluginEntry]]:
         """
-        Initializes and returns the reader (and plugin if relevant) for the provided
-        image based on provided args and/or the available bioio supported plugins
+        Initialize and return the underlying reader (and its PluginEntry, if any)
+        for the provided image.
+
+        Behavior
+        --------
+        * If ``reader`` is a single Reader subclass, that reader is used directly
+          and the plugin system is bypassed.
+
+        * If ``reader`` is a sequence of Reader subclasses, it is treated as a
+          priority list passed to the plugin system. Readers earlier in the list
+          are tried first when multiple plugins support the same extension.
+
+        * If ``reader`` is ``None``, the default plugin ordering is used.
         """
-        if reader is not None:
-            # Check specific reader image types in a situation where a specified reader
-            # only supports some of the ImageLike types.
-            if not check_type(image, reader):
+
+        # Force specific reader
+        if isinstance(reader, type):
```

Comment:
what is `type` in this statement? I can't see where it comes from.

Reply:
I had to google to figure out what this means in Python but now I understand.

Reply:
[20cb352](https://github.com/bioio-devs/bioio/pull/162/commits/20cb35270339337ed95750963e5198fba85bd437)